tests/integration/rsi/test_rsi_submission_data.py

```diff
@@ -1,178 +0,0 @@
-from tests.integration.integration_test_case import IntegrationTestCase
-
-
-class TestRsiSubmissionData(IntegrationTestCase):
-
-    def test_submission_data_1_0102(self):
```

Comment:
Do we need a replacement test to cover submission data? Looks like it was only tested as part of rsi/mci.

Reply:
It should be tested as part of `test_convert_payload_0_0_1.py`